pyproject.toml

```diff
@@ -1,7 +1,7 @@
 [project]
 name = "shiver"
 description = "Spectroscopy Histogram Visualizer for Event Reduction"
-dynamic = ["version"]
+dynamic = ["version", "readme","scripts","license"]
```

Comment:
My understanding is that the dynamic information is only the things that aren't in the repo directly (like version number, because it is generated from tags). The readme, scripts, and license are in git.

Reply:
I added them there, because I was getting some warnings (https://github.com/neutrons/Shiver/actions/runs/5978694811/job/16221271254#step:5:466) during conda build.

![image](https://github.com/neutrons/Shiver/assets/3339090/fe0a627f-6e47-49ba-9d06-7c1a7f15d106)

I can omitt them, if it creates an issue somewhere else though.

Reply:
It is because that information (I verified license is is still in the `setup.cfg` "metadata" section). The warning is to let you know you are being potentially inconsistent. Setuptools is moving towards getting rid of `setup.cfg` so moving things from there to `pyproject.toml` is working for the future.